website/i18n/zh-CN/code.json

```diff
@@ -0,0 +1,198 @@
+{
+  "theme.ErrorPageContent.title": {
+    "message": "This page crashed.",
+    "description": "The title of the fallback page when the page crashed"
+  },
+  "theme.ErrorPageContent.tryAgain": {
+    "message": "Try again",
+    "description": "The label of the button to try again when the page crashed"
+  },
+  "theme.NotFound.title": {
+    "message": "页面找不到啦",
+    "description": "The title of the 404 page"
+  },
```

Comment:
This is the only translation I manually filled in, the other ones are either the same with the English parts or already filled in third-party code.
have a look at these arcane oriental runes:
https://yew-rs--pr2349-code-translation-flo-bk98mfox.web.app/zh-CN/404

Reply:
arcane indeed!